weaviate/tests/utils.py

```diff
@@ -0,0 +1,12 @@
+# (C) Datadog, Inc. 2022-present
```

Comment:
```suggestion
# (C) Datadog, Inc. 2023-present
```

😉 

![what-year-is-it-robin-williams](https://github.com/DataDog/integrations-core/assets/1266346/80a1c4fb-78c8-4db3-9461-83ca21520e87)